src/views/accounts/AccountDiscover/DeleteDialog.vue

```diff
@@ -85,6 +85,10 @@ export default {
     defaultDeleteAccount: {
       type: Boolean,
       default: false
+    },
+    closeable: {
+      type: Boolean,
+      default: true
     }
   },
   data() {
```

Comment:
The provided code seems to be incomplete, missing the relevant context (i.e., what it is intended to do). Please provide additional information so I can analyze the code accurately.